core/src/main/java/io/parsingdata/metal/token/Token.java

```diff
@@ -68,6 +68,8 @@ public boolean isLocal() {
         return true;
     }
 
+    public Token getDefinition(final Environment env) { return this; }
```

Comment:
Not sold yet on the name of this method. Until now, the term "definition" only had meaning inside a `ParseGraph`, where each item is *defined* by a `Token`. This might confuse us in the future.

Not sure what to call this one, though... For lack of a better name, I guess this will have to do.